addons/mail/controllers/mail.py

```diff
@@ -122,6 +122,9 @@ def _redirect_to_record(cls, model, res_id, access_token=None, **kwargs):
             'active_id': res_id,
             'action': record_action.get('id'),
         }
+        menu_id = request.env['ir.ui.menu']._get_best_backend_root_menu_id_for_model(model)
```

Comment:
Side question: if we have an action with a path, shouldn't we send a link using the path instead of the old crafted URL ?

Reply:
As discussed, we will keep the links using the controller “/mail/view” in the emails. Actually, that link is embedded in a firebase redirection (in mail_mobile) so the URL the client receives is anyway complex. Note that “_redirect_to_record” still use “/web” old style URL but that URL is converted automatically to “/odoo” style.